File: dragon_go_countdown/src/main/java/net/tevp/dragon_go_countdown/authentication/DragonAuthenticatorService.java
package net.tevp.dragon_go_countdown.authentication;

import android.app.Service;
import android.content.Intent;
import android.os.IBinder;

public class DragonAuthenticatorService extends Service {
    @Override
    public IBinder onBind(Intent intent) {
        DragonAccountAuthenticator authenticator = new DragonAccountAuthenticator(this);
        return authenticator.getIBinder();
    }
}

File: dragon_go_notifier/src/main/kotlin/net/tevp/dragon_go_notifier/authentication/DragonAuthenticatorService.kt
package net.tevp.dragon_go_notifier.authentication

import android.app.Service
import android.content.Intent
import android.os.IBinder

class DragonAuthenticatorService : Service() {
    override fun onBind(intent: Intent): IBinder {
        val authenticator = DragonAccountAuthenticator(this)
        return authenticator.iBinder
    }
}
